cli/consts.go

```diff
@@ -8,6 +8,7 @@ var (
 	Version        = "TBD"
 	BuildTimestamp = "BuildTimestamp is not set"
 	DownloadURL    = "URL not set yet"
+	CallbackPorts  = []string{"57468", "58888", "59999", "60000"}
```

Comment:
Do you know how many ports Okta allows? I am wondering if we can add more if that makes sense.

Reply:
I'm not sure but I want to try to increase this to 16 ports that are non-contiguous from the 40 to 50k range